lib/ipfs/RequestHandler.ts

```diff
@@ -56,7 +56,7 @@ export default class RequestHandler {
    * Handles read request
    * @param base64urlEncodedMultihash Content Identifier Hash.
    */
-  public async handleFetchRequest (base64urlEncodedMultihash: string, maxSizeInBytes?: number): Promise<ResponseModel> {
+  public async handleFetchRequest (base64urlEncodedMultihash: string, maxSizeInBytes?: number, retryCount?: number): Promise<ResponseModel> {
```

Comment:
Design: Not a fan of the recursive usage here. Retry is an internal implementation detail, can we hide it internally, like using a loop or something?